cmd/algorand-indexer/daemon.go

```diff
@@ -25,17 +27,26 @@ var (
 	noAlgod          bool
 	developerMode    bool
 	allowMigration   bool
+	metricsMode      string
 	tokenString      string
 )
 
+var importTimeHistogramSeconds = prometheus.NewSummary(
```

Comment:
Comment on what this does? It registers something into a package level global and then disappears.

Reply:
done